3130/1/gem_info.rb

```diff
@@ -0,0 +1,84 @@
+require 'open-uri'
+require 'json'
+require 'nokogiri'
+
+class GemInfo
+  WATCHER_WEIGHT = 0.5
+  USED_BY_WEIGHT = 1
+  STAR_WEIGHT = 0.7
+  CLOSE_ISSUE_WEIGHT = 0.9
+  OPEN_ISSUE_WEIGHT = -0.3
+
+  def initialize(gem_names)
+    @gem_names = gem_names
+    @gem_data = []
```

Comment:
Not needed here. Make a method which returns this data. Use memoization if you need it more than once and the operation is slow